index.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+		<meta charset="UTF-8">
+		<meta name="viewport" content="width=device-width, initial-scale=1.0">
+		<title>Web Component Tutorial</title>
+		<script src="user-avatar.js" defer></script>
+		<script src="live-timer.js" defer></script>
+		<script src="word-counter.js" defer></script>
+	</head>
+	
+	
+	<body>
+		<div>
+			<h2>Task #1.</h2>
+			<user-avatar
+				avatar="https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcR51IEd0e1aBFElkIFiSFFJImFt5wENe-9yqw&usqp=CAU">
+			</user-avatar>
+			<user-avatar
+				avatar="https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcQm6L2jdgUcljnfqT2QraTGxE3rXXZGDk-SFQ&usqp=CAU">
+			</user-avatar>
+		</div>
+		<div>
+			<h2>Task #2.</h2>
+			<live-timer id="live-timer-id"></live-timer>
+		</div>
+		<div>
+			<h2>Task #3.</h2>
+			<word-counter></word-counter>
```

Comment:
send to word-counter the text via attribute "text"
```suggestion
			<p is="word-counter" text="Acesta este un text simplu. Cate cuvinte am?"></p>
```